mautic/mautic
# Prevent track contact just by embed form

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Yes
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/6034
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Based on this conversations https://github.com/mautic/mautic/issues/6034,  the pull request prevent create contact (https://github.com/kuzmany/mautic/blob/d039cb16e04aadecd996b733806577d0eefeecb8/app/bundles/FormBundle/Model/FormModel.php#L806)  until you create first field with Auto fill data option

#### Steps to reproduce bug
Steps to reproduce:
1. Create form and call mauticinstance/form/generate.js?id=xx
2. go on contacts > anonymous
3. see anonymouse  contact here with your IP

#### Steps to test this PR:
1.  Repeat all steps
2. See If anonymouse contact is not tracked until you create form field with enable Auto fill data
